test/utils/membership-test.ts

```diff
@@ -17,8 +17,37 @@ limitations under the License.
 import { MatrixClient, MatrixEvent, Room, RoomMember, RoomState, RoomStateEvent } from "matrix-js-sdk/src/matrix";
 import { mocked } from "jest-mock";
 
-import { waitForMember } from "../../src/utils/membership";
-import { createTestClient } from "../test-utils";
+import { isKnockDenied, waitForMember } from "../../src/utils/membership";
+import { createTestClient, mkRoomMember, stubClient } from "../test-utils";
+
+describe("isKnockDenied", () => {
+    // const userId = MatrixClientPeg.get()!.getSafeUserId() //TODO: MA
```

Comment:
There is still a TODO comment here. Hard-coding any matrix ID should be okay here.